android/src/main/java/com/adyenreactnativesdk/component/base/BaseModule.kt

```diff
@@ -74,8 +63,14 @@ abstract class BaseModule(context: ReactApplicationContext?) : ReactContextBaseJ
             .emit(DID_FAILED, ReactNativeError.mapError(error))
     }
 
-    private fun sendFinishEvent(result: SessionDetailsResponse) {
-        val jsonObject = SessionDetailsResponse.SERIALIZER.serialize(result)
+    private fun sendFinishEvent(result: SessionPaymentResult) {
+        val jsonObject = JSONObject().apply {
+            put("resultCode", result.resultCode)
```

Comment:
Can this be a constant?

Reply:
Fixed [7627dd0](https://github.com/Adyen/adyen-react-native/pull/387/commits/7627dd02afebb4d2f43aba68776159ec49146db0)